src/Layout.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Layout.css';
+
+const Layout = () => {
+	return (
+		<div className="layout">
+			<div className="navbar">Navbar (logo + menu button)</div>
+			<div className="page_content">
+				<h1>Main section</h1>
+			</div>
+			<div className="footer">Footer (secondary navigation?)</div>
```

Comment:
Nie wiem, czy mogłoby tak być w ogóle, więc wrzucam jako pomysł - może footer powinien być naszym "main_navbar", a ten górny "secondary_navbar"? Jeśli pomysł byłby zaakceptowany, to wtedy w pliku css zmieniłabym w grid nazwy areas odpowiednio.

Reply:
Nie wiem też. Póki co chyba bym zostawiła jak jest.